apps/api/src/app/widgets/e2e/initialize-widget-session.e2e.ts

```diff
@@ -2,7 +2,7 @@ import { UserSession } from '@notifire/testing';
 import * as jwt from 'jsonwebtoken';
 import { expect } from 'chai';
 
-describe('Initialize Session - /widgets/session/initialize (POST)', async () => {
+describe.only('Initialize Session - /widgets/session/initialize (POST)', async () => {
```

Comment:
Could you please remove the `.only` so we can have our test suite running :)

Reply:
Yeah I removed it now sorry about that :)